### ISSUE TITLE
"WASM Variant" link is dead

### ISSUE THREAD

**Issue by coder543** [anowell/quasar#4]
On [this page](https://anowell.github.io/quasar/), it links to [this page](https://anowell.github.io/wasm.html) which doesn't exist.

**Comment by anowell** [comment]
Indeed. "Coming soon" hasn't been a bit misleading as I haven't done any WASM work yet. I just shook off a few massive time sinks that kept me from Quasar for the last few months, so I'm just now picking up where I left off, which is I'm in the middle of a massive refactor to possibly leverage stdweb. I have much more clarity on the API I want to surface, so as soon as that feels usable, I definitely want to switch gears and focus on WASM as there is a lot of good community momentum around WASM right now.
